files/eni-max-pods.txt

```diff
@@ -101,6 +101,7 @@ p2.16xlarge 234
 p3.2xlarge 58
 p3.8xlarge 234
 p3.16xlarge 234
+p3dn.24xlarge 737
```

Comment:
whats the number you have used for below expression
# of ENI * (# of IPv4 per ENI - 1)  + 2

Reply:
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#AvailableIpPerENI
Based on the documentation, it could have up to 15 interfaces and each with 50 IpV4 address. 
15 * (50 - 1) + 2 = 737